pkg/server/endpoints/node/handler.go

```diff
@@ -157,9 +157,16 @@ func (h *Handler) FetchSVID(server node.Node_FetchSVIDServer) (err error) {
 			return errors.New("Error trying sign CSRs")
 		}
 
+		bundle, err := h.getBundle()
+		if err != nil {
+			h.Log.Errorf("Error retreiving bundle from datastore: %v", err)
+			return fmt.Errorf("Error retreiving bundle")
```

Comment:
nit: error strings should not start with a capital letter.
https://github.com/golang/go/wiki/Errors

Reply:
yesss 😢  I initially had this as a more idiomatic error, but it stood out pretty badly as all the rest of the returned errors in this handler are written like this. In this case, I took consistency over correctness, hoping that we will refactor all of this soon, and use proper gRPC error codes.

Gonna go ahead and merge this since the feedback is labeled as a nit. Feel free to revert if you'd rather see it done correctly.